chapter_convolutional-modern/batch-norm.md

```diff
@@ -448,6 +575,12 @@ net[1].gamma.reshape((-1,)), net[1].beta.reshape((-1,))
 
 ```
 
+```{.python .input}
+#@tab tensorflow
+net.get_layer(index=1).gamma,net.get_layer(index=1).beta
```

Comment:
`net.layers[1]` is more consistent with other implementations

Reply:
got it，thank you